lib/datadog/ci/contrib/minitest/test.rb

```diff
@@ -27,17 +28,25 @@ def before_setup
               end
 
               test_suite_name = Helpers.test_suite_name(self.class, name)
-              source_file, line_number = method(name).source_location
+
+              test_method = method(name)
+              source_file, first_line_number = test_method.source_location
+              last_line_number = Utils::SourceCode.last_line(test_method)
+
+              # @type var tags : Hash[String, String]
+              tags = {
+                CI::Ext::Test::TAG_FRAMEWORK => Ext::FRAMEWORK,
+                CI::Ext::Test::TAG_FRAMEWORK_VERSION => datadog_integration.version.to_s,
+                CI::Ext::Test::TAG_SOURCE_FILE => Git::LocalRepository.relative_to_root(source_file),
+                CI::Ext::Test::TAG_SOURCE_START => first_line_number.to_s
+              }
```

Comment:
This is not really introduced in this PR, but it occured to me that `#source_location` can be nil sometimes, e.g.:

```
[].method(:each).source_location
 => nil 
```

...in practice it shouldn't happen on a test suite, but mentioning in case you may want to do a bit more defensive coding around this case?